### PR TITLE
Dicom support

### PR DESCRIPTION
**Pull request by sameeul**
Initial commit for for processing grayscale dicom files in Nyxus (both single frame and whole-slide image)

- Added NyxusGrayScaleDicomLoader class
- Modified CMakeLists.txt for including DCMTK
- Added steps to build DCMTK and its dependencies in install_prereq_* scripts
- Updated workflows to build wheels 
- Added dcmtk as a dependency for conda build

### CODE REVIEW

**Review by friskluft — COMMENTED** [review]
Good PR overall except one thing: I didn't find the point of defining preprocessor symbol 'DICOM_SUPPORT', probably overlooked I it. Samee please shed the light.
=A

File: CMakeLists.txt [hunk 1]
```diff
@@ -217,6 +217,22 @@ else ()
...
+find_package(ZLIB REQUIRED)
+if (ZLIB_FOUND)
+	list(APPEND Nyxus_LIBRARIES ${ZLIB_LIBRARIES} )
+	include_directories (${ZLIB_INCLUDE_DIR})
+endif (ZLIB_FOUND)
+
+find_package(DCMTK)
+if(DCMTK_FOUND)
+	include_directories(${DCMTK_INCLUDE_DIRS})
+	add_definitions(-DOMEZARR_SUPPORT)
```
[the remark this answers is not in the document]
Comment (sameeul) [reply CMakeLists.txt:229]: Thanks for finding it. Will update.

**Comment by sameeul** [comment]
Closing it for now to fix the build issue.

**Comment by sameeul** [comment]
Reopened after fixing the build issue.

File: CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (sameeul) [reply CMakeLists.txt]: Updated

**Review by friskluft — APPROVED** [review]

**Pull request merged**